image/image.go

```diff
@@ -119,20 +174,41 @@ func downloadUnpackGadget(sto Store, model *asserts.Model, opts *Options) error
 	return snap.Unpack("*", opts.GadgetUnpackDir)
 }
 
-func acquireSnap(sto Store, snapName string, dlOpts *downloadOptions) (downloadedSnap string, info *snap.Info, err error) {
-	// FIXME: add support for sideloading snaps here
-	return downloadSnapWithSideInfo(sto, snapName, dlOpts)
+func acquireSnap(sto Store, name string, dlOpts *downloadOptions, local *localInfos) (downloadedSnap string, info *snap.Info, err error) {
+	if info := local.Info(name); info != nil {
+		// local snap to sideload
```

Comment:
I guess this is not necessarily side-loading, strictly speaking?  Well.. or it depends on what side-loading means I guess. If we have the assertions, this will be just as good as a snap downloaded from the store, right?